app/models/item.rb

```diff
@@ -71,6 +71,18 @@ def remove_from_waitlist(user)
     waitlist.remove_user(user)
   end
 
+  def status_pending_pickup?
+    lend_status == "pending_pickup"
+  end
+
+  def perform_pickup_check
+    return unless status_pending_pickup?
+
+    set_status_available
+    self.holder = nil
+    save
```

Comment:
Am I missing something or should there be two reminders: one real reminder that tells the borrower to remember to pick their item up and a second one that triggers this code (setting the status to available). Maybe I'm misinterpreting the acceptance criteria.

Reply:
yes, this is quite confusing. 
The two notifications belong to issues 158 and 159, but I had to talk to other people first to find out.

Reply:
Oh that is indeed confusing.